src/createNavigationContainer.js

```diff
@@ -160,16 +175,7 @@ export default function createNavigationContainer<T: *>(
       }
       const nav = Component.router.getStateForAction(action, state.nav);
```

Comment:
Good point, worth refactoring when as we work on it. I'll get back to that later today.